CanonicalLtd/landscape-client
# Add option to stagger launch of broker plugins.

Testing instructions:
sudo ./scripts/landscape-config --stagger-launch 0.5 -n -c root-client.conf
cat root-client.conf | grep stagger
sudo ./scripts/landscape-client --log-level debug -c root-client.conf
Check for debug log entries mentioning monitors getting delayed
